### PR TITLE
Release 1.4.2.5

### PR DESCRIPTION
**Pull request by abernix**
During the final stages of the release process for Meteor 1.4.2.4 (a bug fix release, #8306) it was realized that it was going to be necessary for developers to take the additional, possibly mandatory, step of running `meteor update --all-packages` or `meteor update modules-runtime`.  This is because Meteor 1.4.2.4 requires `modules-runtime@0.7.8` and its dependency on (NPM) `install@0.8.4` to function correctly.  See [this commit](https://github.com/benjamn/install/commit/377d1a3b5138904d63f4594fd4c12f756454d277) on [`install`](https://github.com/benjamn/install) or #8213 if you're interested in the logic behind that (important!) change.

Despite the simple solution (stated above) and while this was explained very clearly in the release notes, it was deemed too complex for this point release due to the additional intervention necessary by the developer.  Meteor 1.4.2.4 was not (and will not be) marked as `recommended` (i.e. made official).

In an ideal world `meteor-tool` could have forced a minimum version for this Atmosphere package to `0.7.8` but due to version unpinning this update would not have been enforced by the `constraint-solver`.  This will likely be addressed in the future.

This release will simply omit 2eab0b2bf4d635c0a7d8b3f2b1569e03b73a4325 (and associated changes) although those will remain in the upcoming 1.4.3 (#8123) release with the addition of an "upgrader" designed to take care of this automatically with no additional steps required by the end-developer.

### CODE REVIEW

File: packages/modules-runtime/modules-runtime.js [hunk 1]
```diff
@@ -13,10 +13,6 @@ if (typeof Profile === "function" &&
   };
 }
 
-// On the client, make package resolution prefer the "browser" field of
-// package.json files to the "main" field.
-options.browser = Meteor.isClient;
```
Comment (benjamn) [inline packages/modules-runtime/modules-runtime.js]: @abernix Let's not revert this change?

Comment (abernix) [inline packages/modules-runtime/modules-runtime.js]: @benjamn Yes, it shouldn't be reverted and the commit above is actually deceiving as I didn't end up publishing `modules-runtime` in the release process for `rc.0`.  But this should accurately reflect what was published.  I'll fix this.

**Comment by mitar** [comment]
Hm, why I am getting that `1.4.2.5` version is available, but there is nothing available in `History.md` in `devel` branch about it?

**Comment by abernix** [comment]
@mitar You can find the `History.md` for 1.4.2.5 on the [`release-1.4.2.5`](https://github.com/meteor/meteor/blob/release-1.4.2.5/History.md) branch.  We now have the concept of more than one active release branch in order to release bug fixes at a faster velocity and independent of a more substantial release.

Obviously, we still need to figure out what exactly that means for the `devel` branch.  `History.md` seems like the most important thing to get updated at the moment though, and the recent inclusion of this in [the docs](http://docs.meteor.com/changelog.html) might be the solution for that.  It wasn't updated when the 1.4.2.5 was officially recommended, but it is [updated now](http://docs.meteor.com/changelog.html#v1425).

**Comment by mitar** [comment]
Yes, it is merged now, because this pull request got merged into `deve`. ;-)

BTW, one cool thing could be to display the segment of history.md file in the console. So one could see there what is new. Instead of having to go to the GitHub.
